coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java

```diff
@@ -206,7 +205,16 @@ else if (isDisapproved(statusChangeEvent)) {
         }
     }
     
+    protected DocumentRouteHeaderValue getDocumentRouteHeaderValue() {
+        return KcServiceLocator.getService(RouteHeaderService.class).getRouteHeader(
+                this.getDocumentHeader().getWorkflowDocument().getDocumentId());
+    }
   
+    protected void setProtocolDocumentToApproveByDefault() {
+        DocumentRouteHeaderValue document = getDocumentRouteHeaderValue();
+        document.getDocumentType().getDefaultApprovePolicy().setPolicyValue(true);
+    }
```

Comment:
@rmancher this has always been false for protocol right? Why is this change needed now?